app/src/main/java/com/vestrel00/daggerbutterknifemvp/ui/common/BaseFragment.java

```diff
@@ -53,13 +58,52 @@
     @Inject
     DispatchingAndroidInjector<Fragment> fragmentInjector;
 
+    @Nullable
+    private Unbinder viewUnbinder;
+
     @Override
     public void onAttach(Context context) {
         AndroidInjection.inject(this);
         super.onAttach(context);
     }
 
     @Override
+    public void onViewStateRestored(Bundle savedInstanceState) {
+        super.onViewStateRestored(savedInstanceState);
+        View view = getView();
+        if (view != null) {
```

Comment:
Null check not needed since `onViewStateRestored` only gets called when the `View` returned in `onCreateView` is not null. See https://github.com/vestrel00/android-dagger-butterknife-mvp/commit/7370d995c8cd6e946e94c30c6bf4537cb4c34aab